size_benchmarks/earlyflush/sizes.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+import argparse
+import concurrent.futures
+import glob
+import itertools
+import io
+import os
+import subprocess
+
+
+# binjs_generate_prediction_tables takes a dictionary output path
+# whereas binjs_encode takes an input dictionary file; we just work
+# with paths and hard-code the filename the same way as
+# binjs_generate_prediction_tables.
+def dict_path_to_filename(path):
+  return os.path.join(path, 'dict.entropy')
+
+
+def generate_dictionary(args):
+  input_files = args.input_files
+  output_dictionary = args.dictionary
+  cmd = ['../../target/release/binjs_generate_prediction_tables',
```

Comment:
This looks like a dependency on the optimized version. Maybe build it yourself or use `cargo run` to launch it, ensuring that it's always built?

Here and below.